src/drivers/code_hosting/activate.sh

```diff
@@ -14,6 +14,10 @@ function activate_driver_for_code_hosting {
     activate_driver 'code_hosting' 'github'
   elif [ "$origin_hostname" == 'bitbucket.org' ]; then
     activate_driver 'code_hosting' 'bitbucket'
+  elif [ "$(contains_string "$origin_hostname" github)" == true ]; then
```

Comment:
Thanks for the issue. What I was referring to though was documenting on "git new-pull-request" / "git-repo" that if you are using ssh identity that points to github/bithucket, the identity name needs to include github/bitbucket